gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/InMemoryGoogleCloudStorage.java

```diff
@@ -109,7 +111,7 @@ private boolean validateObjectName(String objectName) {
 
   @Override
   public synchronized WritableByteChannel create(StorageResourceId resourceId) throws IOException {
-    return create(resourceId, CreateObjectOptions.DEFAULT);
+    return create(resourceId, CreateObjectOptions.DEFAULT_OVERWRITE);
```

Comment:
I guess here is an example of the scenario I mentioned  above - where a DEFAULT option is required but the overwrite one is used.

Reply:
`DEFAULT` was overwrite option in previously, I just make it explicit.